src/components/filterprovider.tsx

```diff
@@ -1,11 +1,15 @@
-import React, { useReducer, useContext, createContext } from "react";
+import React, { useEffect, useReducer, useContext, createContext } from "react";
 
 import { IncidentsFilter, AutoUpdate } from "../components/incidenttable/FilteredIncidentTable";
 import { Filter } from "../api";
 import { Tag, originalToTag } from "../components/tagselector";
 
 // Store
 import { ActionMap } from "../reducers/common";
+import { SELECTED_FILTER } from "../localstorageconsts";
```

Comment:
There's no no complaint here so I guess `IncidentFilterToolbar.tsx` is processed before `filterprovider,tsx`.